frontend/src/routes/Matches/Match/Match.css

```diff
@@ -3,4 +3,11 @@
     grid: auto-flow 2rem / repeat(3,3rem);
     align-items: center;
     justify-items: center;
+}
+
+.justRow {
```

Comment:
kabob case: `just-row`

Reply:
i can also just delete this, right?

Reply:
Yea it does not seem to be used, so it can be deleted